LBHackney-IT/lbh-fss-portal-frontend
# Feature/VCSO analytics page

**Description**

Add analytics page for VCSO users. This should include a total number of views per service, and should be able to filter by date. 

**Test instructions** 

- pull this branch 
- `yarn install`
- navigate to **/analytics**
- numbers should update on date filter, if trying to search for an invalid date range (start date after end date, or vice-versa) should get a validation error
- switch user types by navigating to /mock-api/account/GET.js
    - ADMIN: should see a different analytics page (containing overview numbers)
    - NO services but WITH organisation: should see 'no services' message 
    - WITH services but WITHOUT organisations: should not be able to see the analytics tab in the navbar, and should get Access Denied message if trying to navigate directly to **/analytics**
- also try seeing what happens when either no data or a 400 is sent from the api by editing /mock-api/analytics-event/GET.js

**Task**
https://app.clickup.com/t/a6zekz
